src/core/components/AMInstallButton/index.js

```diff
@@ -397,13 +395,7 @@ const AMInstallButton: React.ComponentType<Props> = compose(
   withRouter,
   connect(mapStateToProps),
   translate(),
-  withExperiment({
-    id: EXPERIMENT_ID,
-    variants: [
-      { id: VARIANT_INCLUDE_WARNING, percentage: 0.5 },
-      { id: VARIANT_EXCLUDE_WARNING, percentage: 0.5 },
-    ],
-  }),
+  withExperiment(EXPERIMENT_CONFIG),
```

Comment:
I know this isn't a change introduced by your patch but why does this share the same experiment config with `InstallWarning`? Isn't the experiment self-contained within `InstallWarning`?

Reply:
It is not. We also need to send an event when someone clicks on the install button, which is why both components need access to the experiment.

Reply:
Aha, it's for the actual install click. That's the concept I was missing. Thanks.